docs/it-manual/api/program-applications.md

```diff
@@ -606,19 +607,25 @@ In addition to the metadata field, name questions have the following properties:
 - **Property**: `first_name`
 - **JSON Type**: `string` or `null`
 - **Format**: A Unicode string of any characters. `null` if the question is unanswered.
-- **Description**: The applicant's first name. If the question is answered, only the `middle_name` is optional.
+- **Description**: The applicant's first name. If the question is answered, only the `middle_name` and `suffix` are optional.
 
 #### `middle_name`
 - **Property**: `middle_name`
 - **JSON Type**: `string` or `null`
 - **Format**: A Unicode string of any characters. `null` if unanswered or not provided.
-- **Description**: The applicant's middle name. If the question is answered, only the `middle_name` is optional.
+- **Description**: The applicant's middle name. If the question is answered, only the `middle_name` and `suffix` are optional.
 
 #### `last_name`
 - **Property**: `last_name`
 - **JSON Type**: `string` or `null`
 - **Format**: A Unicode string of any characters. `null` if the question is unanswered.
-- **Description**: The applicant's last name. If the question is answered, only the `middle_name` is optional.
+- **Description**: The applicant's last name. If the question is answered, only the `middle_name` and `suffix` are optional.
+
+#### `suffix`
+- **Property**: `suffix`
+- **JSON Type**: `string` or `null`
+- **Format**: A Unicode string of any characters. `null` if unanswered or not provided.
```

Comment:
Isn't this restricted to a few specific options? If so, we should document this similar to the way we do for `revision_state`.

Reply:
Yeah it would be better to mention that. Added it!